docs/user-guide/dual-mesh.ipynb

```diff
@@ -0,0 +1,6253 @@
+{
```

Comment:
<p>I'd suggest elaborating more on what the "dual" means in the broader context of the geometry. Feel free to rephrase / incorporate any of the following</p><ul><li>When computing the dual of a grid, the original grid is typically referred to as the "Primal" grid. The corner nodes of the Primal Grid become the face centers of the Dual Grid, and the face centers of the Primal Grid become the corner nodes of the Dual Grid.</li><li>This means that variables that were originally face centered on the Primal Grid will be node centered on the Dual grid, and vice versa. </li></ul><p> </p><p>The warning about global dual meshes should be turned into an HTML warning admonition. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Ok! I did remove the method parameter from the code, as there doesn't seem to be a need for two different methods. At first I thought there was, but it seems I can do all that inside the bilinear code, so I will remove that from the notebook.</p>